src/client/auth/auth.go

```diff
@@ -213,3 +236,25 @@ func IsErrTooShortTTL(err error) bool {
 		strings.Contains(errMsg, ") is shorter than token's existing TTL (") &&
 		strings.Contains(errMsg, ")")
 }
+
+// HashToken converts a token to a cryptographic hash.
+// We don't want to store tokens verbatim in the database, as then whoever
+// that has access to the database has access to all tokens.
+func HashToken(token string) string {
+	sum := sha256.Sum256([]byte(token))
+	return fmt.Sprintf("%x", sum)
+}
```

Comment:
Since `Activate()` takes a root token instead of a hash now, do you think it would make sense to keep this in the server? I guess I worry a little that it'll be harder to change if it's in the client package (though if it's not in our API, maybe not).